### PR TITLE
Implemented SuccessPage and routing for form submission

### PR DESCRIPTION
**Pull request by Palakmeena**
I request a thorough review as I want to avoid any issues that might arise from merging this without proper validation.

### CODE REVIEW

File: src/components/SuccessPage.tsx
Comment (zealmaster) [inline src/components/SuccessPage.tsx:1]: Move the ccs styles to index.css

File: src/index.tsx
Comment (zealmaster) [inline src/index.tsx:1]: Remove the components import

File: src/components/SuccessPage.tsx
Comment (zealmaster) [inline src/components/SuccessPage.tsx:1]: Move the ccs styles to App.css

**Review by zealmaster — CHANGES_REQUESTED** [review]
Move the ccs styles to App.css
Remove the components import

**Comment by Palakmeena** [comment]
I completed the redirecting part , please review it

**Review by zealmaster — APPROVED** [review]

**Review by Mo-wo — APPROVED** [review]

**Pull request merged**